lib/Runtime/Language/InlineCache.h

```diff
@@ -513,10 +513,6 @@ namespace Js
         template<bool CheckLocal, bool CheckProto, bool CheckAccessor>
         void CloneInlineCacheToEmptySlotInCollision(Type *const type, uint index);
 
-#ifdef CLONE_INLINECACHE_TO_EMPTYSLOT
```

Comment:
>CLONE_INLINECACHE_TO_EMPTYSLOT [](start = 7, length = 30)

Please dont delete this code. There has been an instance where we ended up using some code under this ifdef. The code under this ifdef is still good to keep around for experimenting with it.